example.config.json

```diff
@@ -1,6 +1,7 @@
 {
     "prover_name_prefix": "sindri_",
     "keys_dir": "keys",
+    "db_path": "db",
```

Comment:
The key and suggested value here were obtained directly from [upstream](https://github.com/scroll-tech/scroll-proving-sdk/pull/49/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947). I'd prefer compatibility over something more verbose